test/main.test.js

```diff
@@ -234,6 +235,10 @@ describe('main', () => {
         0,
         "Player #1's jail status was not correctly set"
       );
+      expect(gameState.players[0].cash).to.equal(
+        1500 - config.incomeTaxAmount,
```

Comment:
Wondering about popping this `1500` into a helper constant for the scope of this file as well